easybuild/easyblocks/generic/conda.py

```diff
@@ -79,7 +83,6 @@ def install_step(self):
 
             self.set_conda_env()
 
-            # use --force to ignore existing installation directory
             cmd = "%s conda env create --force %s -p %s" % (self.cfg['preinstallopts'], env_spec, self.installdir)
```

Comment:
Its usage is rather mysterious to me. I think it means that it will create the env even if one or more of the install packages fails.